cpp/src/arrow/array/builder_binary.h

```diff
@@ -189,21 +189,27 @@ class ARROW_EXPORT FixedSizeBinaryBuilder : public ArrayBuilder {
     return byte_builder_.Append(value, byte_width_);
   }
 
+  void UnsafeAppend(const uint8_t* value) {
```

Comment:
Side note: it would be nice if safe and unsafe variants were grouped together in declaration order.

Reply:
I'll reorder them.